external/emulation/emulation/_filesystem.py

```diff
@@ -10,6 +10,5 @@
 def get_dir(path: str):
     with tempfile.TemporaryDirectory() as tmpdir:
         fs, _, _ = fsspec.get_fs_token_paths(path)
-        # fsspec places the directory inside the tmpdir, as a subdirectory
```

Comment:
This information isn't obvious from fsspec's function name and isn't present in their documentation, so it may be helpful to keep it.

Reply:
sorry...i didn't intended to commit this...was related to another PR.

Reply:
reverted.